tokio-threadpool/src/lib.rs

```diff
@@ -3,6 +3,107 @@
 #![doc(html_root_url = "https://docs.rs/tokio-threadpool/0.1.4")]
 #![deny(warnings, missing_docs, missing_debug_implementations)]
 
+// The Tokio thread pool is a thread pool designed to scheduled futures in Tokio
```

Comment:
yeah, i know the tokio thread pool is a thread pool.

better: The tokio thread pool is designed to schedule futures in Tokio-based applications

(present-tense "schedule", hyphenated "Tokio-based")